ci-operator/config/3scale-qe/3scale-deploy/3scale-qe-3scale-deploy-main__3scale-amp-ocp4.14-lp-rosa.yaml

```diff
@@ -0,0 +1,75 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  test-image:
+    name: 3scale-interop-tests
+    namespace: ci
+    tag: v2.13
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  to: 3scale-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: 3scale-amp-interop-rosa-aws
```

Comment:
I believe this PR is putting in place testing for Rosa Classic but I don't see anything that tells us that explicitly.

We use unique identifiers to help us separate our work

For self-managed we use "-lp-interop"
For Rosa HyperShift we us "-lp-rosa-hypershift"

For Rosa Classic what should we use?

I'd reccomend "-lp-rosa-classic"
What do you think?
This unique identifier can either be in the filename or here in the test stanza name